models/agent.go

```diff
@@ -211,3 +212,43 @@ func (q *QanAgent) DSN(service *MySQLService) string {
 	// TODO Other parameters?
 	return cfg.FormatDSN()
 }
+
+// AgentFilters represents filters for agents list.
+type AgentFilters struct {
+	ServiceID *string
```

Comment:
We don't need to differentiate between `nil` and an empty string (in both cases filter is absent), so no need to use a pointer.